Branch-SDK/Branch-SDK/BNCLinkData.m

```diff
@@ -131,16 +115,6 @@ - (void)setupParams:(NSDictionary *)params {
     }
 }
 
-- (void)setObject:(id)anObject forKey:(id <NSCopying>)aKey {
```

Comment:
Nope, that uses the `BNCLinkCache` methods, which aren't looking for `NSCopying`, just that your item is a `BNCLinkData` item.